pkg/store/filenames/filenames.go

```diff
@@ -91,3 +91,11 @@ const LongestSock = SSHSock + ".1234567890123456"
 func PIDFile(name string) string {
 	return name + ".pid"
 }
+
+// SkipOnClone files should be skipped on cloning an instance.
+var SkipOnClone = []string{
+	Protected,
```

Comment:
I'm pretty sure the VzIdentifier needs to be recreated for the new machine.

```suggestion
	Protected,
	VzIdentifier,
```

Reply:
https://github.com/lima-vm/lima/blob/73c5f9761bff67d9cb746e14a20a8d58bae34b39/pkg/limayaml/defaults.go#L1158-L1164

The VM type will be changed to `qemu` when `VzIdentifier` is removed